spec/components/jobseekers/search_results/heading_component_spec.rb

```diff
@@ -8,6 +8,7 @@
   let(:location) { "London" }
   let(:polygon_boundaries) { [[51.0, 51.0]] }
   let(:count) { 10 }
+  let(:radius) { 1 }
```

Comment:
In this spec we are testing the cases where radius and buffer_radius are both present, but we never test the cases where only one of them is present, which is the more likely scenario.